### ISSUE TITLE
[Bug]: Error message is not output by screenreader

### ISSUE THREAD

**Issue by orsolya-gyovai** [sbb-design-systems/lyne-components#3097]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Operating system

Windows 10

### Browser / Browser version

Chrome Version 128.0.6613.138 (Offizieller Build) (64-Bit)

### Input mode

None

### What happened?

Error message on the input field is not output automatically. 
Error messages should be output automatically, so that blind users are able to perceive them with the screen reader without having to navigate back to the input field again. Error message and input field should be linked, e. g: via Attribut aria-describedby. 
![A11y_2024-09-19_error-message](https://github.com/user-attachments/assets/3bc9ffd0-0c7d-4258-bd2f-350609b8b7fd)

### Steps to reproduce the issue

1. Start NVDA
2. go to the site https://sbb.boncard.ch/balance 
4. Enter an invalid number into the input field, e.g. 12345
5. Click on enter Button

**Comment by kyubisation** [comment]
Hello @orsolya-gyovai 

The linked website does not use it correctly, as it places the `<sbb-form-error>` outside of `<sbb-form-field>`. The  `<sbb-form-error>` should be placed inside of `<sbb-form-field>`, for the automatic assignment of the aria attributes to work.

[Documentation](https://lyne-storybook.app.sbb.ch/?path=/docs/elements-sbb-form-field-sbb-form-field--docs#error-messages)
Example: https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-form-field-sbb-form-field--input-long-label-and-error-space

**Comment by orsolya-gyovai** [comment]
Hi @kyubisation, 
thank you very much for your help and the quick reply. Ah okay... I will forward the information to the relevant team.
thanks again and best regards.
